### PR TITLE
Creating new task; add create/delete feature test

### PR DESCRIPTION
**Pull request by m-ermolaev**
https://user-images.githubusercontent.com/24371248/232778294-3e54c362-fd15-439e-8f3b-d77ed997ce3d.mov

### CODE REVIEW

File: tests/Feature/DeleteTaskTest.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+use App\Models\Task;
+
+class DeleteTaskTest extends TestCase
+{
+    /**
```
Comment (webdevelman) [inline tests/Feature/DeleteTaskTest.php:10]:
```suggestion
    /**use RefreshDatabase;
```

File: tests/Feature/DeleteTaskTest.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+     */
+    public function test_delete_task()
+    {
+        $task = Task::factory()->create([
+            'name' => 'Go to market',
+        ]);
+        $this->assertModelExists($task);
+        $this->assertDatabaseHas('tasks', [
+            'name' => $task->name
+        ]);
```
Comment (webdevelman) [inline tests/Feature/DeleteTaskTest.php]:
```suggestion
       $testTaskName = 'test_name';
        $task = Task::factory()->create([
            'name' => $testTaskName,
        ]);
        $this->assertModelExists($task);
        $this->assertDatabaseHas('tasks', [
            'name' => $testTaskName
        ]);
```

File: tests/Feature/DeleteTaskTest.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        $task->delete();
+        $this->assertDeleted($task);
+        $this->assertDatabaseMissing('tasks', [
+            'name' => $task->name
```
Comment (webdevelman) [inline tests/Feature/DeleteTaskTest.php]:
```suggestion
            'name' => $testTaskName
```

File: app/Http/Livewire/CreateTask.php [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+class CreateTask extends Component
+{
+    public $name;
+
+    public function handleCreateClick()
+    {
+        $task = new Task;
+        $task->name = $this->name;
+        $task->save();
```
Comment (webdevelman) [inline app/Http/Livewire/CreateTask.php]:
```suggestion
        Task::create(['name'=>$this->name]);
```

Comment (webdevelman) [reply app/Http/Livewire/CreateTask.php]: Can we somehow check if the task already exists in the DB and if so, notify the user about it?

**Comment by webdevelman** [comment]
Please add some PR's description.

File: tests/Feature/DeleteTaskTest.php [hunk 5]
```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
```
Comment (webdevelman) [inline tests/Feature/DeleteTaskTest.php]:
```
use Illuminate\Foundation\Testing\RefreshDatabase;
use Illuminate\Foundation\Testing\WithFaker;

```
why do we need these two classes?

File: tests/Feature/DeleteTaskTest.php [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+use Tests\TestCase;
+use App\Models\Task;
+
+class DeleteTaskTest extends TestCase
+{
+    /**use RefreshDatabase;
```
Comment (webdevelman) [inline tests/Feature/DeleteTaskTest.php]: ?

Comment (m-ermolaev) [reply tests/Feature/DeleteTaskTest.php]: Screenshot 2023-04-20 at 01 21 34
I confirmed your suggestion

File: tests/Feature/DeleteTaskTest.php (hunk 5 above)
Comment (m-ermolaev) [reply tests/Feature/DeleteTaskTest.php]: I have already removed it

**Pull request merged**